### PR TITLE
VIDEO-5723 - isSupported for iOS Chrome

### PR DESCRIPTION
**Pull request by PikaJoyce**
JIRA: [VIDEO-5723](https://issues.corp.twilio.com/browse/VIDEO-5723)

Updated support for iOS Chrome and adjusted unit test.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 2.15.0 (In Progress)
 =====================
 
+New Features
+------------
+
+- `Video.isSupported` now returns `true` for iOS Chrome. (VIDEO-5723)
```
Comment (manjeshbhargav) [inline CHANGELOG.md:11]:
```suggestion
- `isSupported` now returns `true` for iOS Chrome. (VIDEO-5723)
```

File: test/unit/spec/util/support.js [hunk 2]
```diff
@@ -33,6 +33,11 @@ describe('isSupported', () => {
       'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/81.0.4044.0 Safari/537.36',
       { runtime: {} }
     ],
+    [
```
Comment (manjeshbhargav) [inline test/unit/spec/util/support.js:36]: Can you add iPad user agent tests for both safari and chrome?

**Review by manjeshbhargav — CHANGES_REQUESTED** [review]
@PikaJoyce Great work! Please re-open the PR against a feature branch please.

File: test/unit/spec/util/support.js (hunk 2 above)
Comment (PikaJoyce) [reply test/unit/spec/util/support.js:36]: There's already a test for iPad Safari UA test, so I'll add the Chrome one!

**Pull request merged**